lib/decidim/civicrm/version.rb

```diff
@@ -3,8 +3,8 @@
 module Decidim
   # This holds the decidim-civicrm version.
   module Civicrm
-    DECIDIM_VERSION = "0.28.3"
-    COMPAT_DECIDIM_VERSION = [">= 0.28.0", "< 0.29"].freeze
+    DECIDIM_VERSION = { github: "decidim/decidim", branch: "release/0.29-stable" }.freeze
+    COMPAT_DECIDIM_VERSION = [">= 0.29", "< 0.30"].freeze
     VERSION = "0.7.0"
```

Comment:
```suggestion
    VERSION = "0.8.0"
```
Let's change version here

Reply:
Done